lib/playground.dart

```diff
@@ -315,21 +341,47 @@ class Playground {
     if (context.focusedEditor == 'dart') {
       ga.sendEvent('main', 'help');
 
+      var input;
       Position pos = editor.document.cursor;
-      var input = new SourceRequest()
+      int offset = editor.document.indexFromPos(pos);
+
+      if (querySelector(".CodeMirror-hint-active") != null) {
```

Comment:
I'm never sure if such getter must be private or not. And shall I add those getters to Playground or to PlaygroundContext ?